### PR TITLE
fix: new header project title 

### PR DESCRIPTION
**Pull request by mortada-codes**
This PR... #3006 

## Changes

- the project title style

## Fixes

-

## How to test it

-

## Screenshots

-

https://user-images.githubusercontent.com/20525304/212888996-73f106c0-2833-418f-b738-e5591da4272a.mp4

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/PageHeader/PageHeader.tsx [hunk 1]
```diff
@@ -34,10 +34,8 @@ import {activeProjectSelector, isInPreviewModeSelector} from '@shared/utils/sele
 import {trackEvent} from '@shared/utils/telemetry';
 
 import ClusterSelection from './ClusterSelection';
-import CreateProject from './CreateProject';
 import {HelpMenu} from './HelpMenu';
 import * as S from './PageHeader.styled';
-import ProjectSelection from './ProjectSelection';
```
Comment (topliceanurazvan) [inline src/components/organisms/PageHeader/PageHeader.tsx:40]: I guess we could also remove these components as they are not used anymore

File: src/components/organisms/PageHeader/PageHeader.styled.tsx [hunk 2]
```diff
@@ -114,3 +119,50 @@ export const Divider = styled(RawDivider)`
...
+  max-width: 272px;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+  color: ${Colors.grey9};
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  text-align: left;
+  transition: all 0.5s ease-in;
```
Comment (topliceanurazvan) [inline src/components/organisms/PageHeader/PageHeader.styled.tsx:152]: we should use the AnimationDurations in here from the shared folder

File: src/components/organisms/PageHeader/PageHeader.styled.tsx [hunk 3]
```diff
@@ -114,3 +119,50 @@ export const Divider = styled(RawDivider)`
...
+`;
+
+export const MenuOutlinedIcon = styled(RawMenuOutlined)`
+  color: ${Colors.grey9};
+  margin-right: 8px;
+
+  svg {
+    width: 20px;
+    height: 20px;
+    transition: all 0.5s ease-in;
```
Comment (topliceanurazvan) [inline src/components/organisms/PageHeader/PageHeader.styled.tsx:166]: Same as above

File: src/components/organisms/PageHeader/PageHeader.styled.tsx [hunk 4]
```diff
@@ -114,3 +119,50 @@ export const Divider = styled(RawDivider)`
...
+    color: inherit !important;
+  }
+`;
+
+export const MenuOutlinedIcon = styled(RawMenuOutlined)`
+  color: ${Colors.grey9};
+  margin-right: 8px;
+
+  svg {
+    width: 20px;
```
Comment (topliceanurazvan) [inline src/components/organisms/PageHeader/PageHeader.styled.tsx:164]: Do we need to set a width and height? Can't we use directly the font-size property on the `MenuOutlinedIcon` component?

File: src/components/organisms/PageHeader/PageHeader.styled.tsx [hunk 5]
```diff
@@ -93,6 +97,7 @@ export const Logo = styled.img`
 
 export const LogoContainer = styled.div`
   height: 100%;
+  min-width: 50px;
```
Comment (topliceanurazvan) [inline src/components/organisms/PageHeader/PageHeader.styled.tsx:100]: Why do we need a min-width if the width is already fixed? 🤔

File: src/components/atoms/TableSelect/TableSelect.tsx [hunk 6]
```diff
@@ -50,10 +50,23 @@ const Button = styled(RawButton)`
...
+      color: ${Colors.geekblue9};
+    }
+  }
+
+  svg {
+    transition: all 0.3s ease-in;
+  }
+
+  span {
+    transition: all 0.3s ease-in;
```
Comment (topliceanurazvan) [inline src/components/atoms/TableSelect/TableSelect.tsx:69]: Use AnimationsDurations

File: src/components/atoms/TableSelect/TableSelect.tsx [hunk 7]
```diff
@@ -50,10 +50,23 @@ const Button = styled(RawButton)`
...
-    color: ${Colors.lightSeaGreen};
+    background-color: transparent;
+
+    span {
+      color: ${Colors.geekblue9};
+    }
+  }
+
+  svg {
+    transition: all 0.3s ease-in;
```
Comment (topliceanurazvan) [inline src/components/atoms/TableSelect/TableSelect.tsx:65]: Use AnimationsDurations

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]

File: src/components/organisms/PageHeader/PageHeader.styled.tsx (hunk 5 above)
Comment (mortada-codes) [reply src/components/organisms/PageHeader/PageHeader.styled.tsx:101]: Because the container flex box pushes the logo back to 40px on small screens

**Comment by topliceanurazvan** [comment]
I am also getting this warning when running the app 

image

I'm guessing it might be because of this

image

**Comment by topliceanurazvan** [comment]
Can we also vertically align these? 
image

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**